mm/2s2h/Enhancements/GfxPatcher/AuthenticGfxPatches.cpp

```diff
@@ -1,7 +1,6 @@
 #include "AuthenticGfxPatches.h"
-#include "libultraship/libultraship.h"
-#include <cstring>
-
+#include "public/bridge/consolevariablebridge.h"
+#include "gfx.h"
 extern "C" {
```

Comment:
Lets move the vanilla source headers into the extern block following our other patterns
```suggestion

extern "C" {
#include "gfx.h"
```